02_configure_host.sh

```diff
@@ -245,7 +245,16 @@ for IMAGE_VAR in $(env | grep "_LOCAL_IMAGE=" | grep -o "^[^=]*") ; do
 
   # Support building ironic-image from source
   if [[ "${IMAGE}" =~ "ironic" ]] && [[ ${IRONIC_FROM_SOURCE:-} == "true" ]]; then
-    sudo "${CONTAINER_RUNTIME}" build --build-arg INSTALL_TYPE=source -t "${!IMAGE_VAR}:latest" -t "${!IMAGE_VAR}:${IMAGE_GIT_HASH}_${IMAGE_DATE}" . -f ./Dockerfile
+    # NOTE(rpittau): to customize the source origin we need to copy the source code we
+    # want to use into the sources directory under the ironic-image repository.
+    for CODE_SOURCE_VAR in $(env | grep -E '^IRONIC_SOURCE=|^IRONIC_INSPECTOR_SOURCE=|^SUSHY_SOURCE=' | grep -o "^[^=]*"); do
```

Comment:
I think this would be far easier to understand if we skip the grep and just enumerate the variables directly. Maybe even skip the loop. It would be slightly more/repeated code but much "simpler".

Just a thought though. I am fine with it this way also and it is in line with what we do in other places in the repo.

Reply:
thanks for the suggestion, maybe in a followup?

Reply:
Sure! :slightly_smiling_face: